ReactNativeClient/android/app/src/main/java/net/cozic/joplin/MainApplication.java

```diff
@@ -39,6 +40,7 @@ public boolean getUseDeveloperSupport() {
 		protected List<ReactPackage> getPackages() {
 			return Arrays.<ReactPackage>asList(
 				new MainReactPackage(),
+            new AppShortcutsPackage(),
```

Comment:
Note to reviewer: I didn't touch this file; these lines were added automatically when I installed `react-native-quick-actions` via npm and linked the project in xcode (as described in the OP of this PR)